da/grpc/mockserv/cmd/.gitignore

```diff
@@ -0,0 +1,2 @@
+cmd
+db
```

Comment:
[non-blocking] Why do we need cmd and db in gitignore ? How is that related to contexts.

Reply:
removed it